src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+public class Deadline extends Task {
```

Comment:
please include documentation for all non-trivial classes and methods

Reply:
Noted